docker/build/assets.Dockerfile

```diff
@@ -171,7 +172,9 @@ RUN cd /cuda-quantum && source scripts/configure_build.sh && \
     "$LLVM_INSTALL_PREFIX/bin/llvm-lit" -v build/test \
         --param nvqpp_site_config=build/test/lit.site.cfg.py && \
     "$LLVM_INSTALL_PREFIX/bin/llvm-lit" -v build/targettests \
-        --param nvqpp_site_config=build/targettests/lit.site.cfg.py
+        --param nvqpp_site_config=build/targettests/lit.site.cfg.py && \
+    "$LLVM_INSTALL_PREFIX/bin/llvm-lit" -v build/python/tests/mlir \
+        --param nvqpp_site_config=build/python/tests/mlir/lit.site.cfg.py
```

Comment:
Python is built separately in its own stage. You will need to copy over the build folder from that stage. Scikit is used to build the python support; the pyproject.toml defines the build and disables building tests. I haven't checked where they would end up in if they are enabled during the build. Give it a go and let me know if you run into trouble.